storage/range_gc_queue.go

```diff
@@ -145,6 +136,16 @@ func (q *rangeGCQueue) process(now proto.Timestamp, rng *Range) error {
 
 		// TODO(bdarnell): remove raft logs and other metadata (while leaving a
 		// tombstone). Add tests for GC of merged ranges.
+	} else {
+		// This range is a current member of the raft group. Acquire the lease
+		// to avoid processing this range again before the next inactivity threshold.
+		if err := rng.requestLeaderLease(now); err != nil {
```

Comment:
yeah, but it doesn't jitter. In any case, will take a while for this to be an issue and then we can fix it.